pfioh/pfioh.py

```diff
@@ -657,8 +660,11 @@ def unpackForm(self, form, d_form):
 
         self.dp.qprint("Unpacking multi-part form message...", comms = 'status')
         for key in form:
-            self.dp.qprint("\tUnpacking field '%s..." % key, comms = 'status')
-            d_form[key]     = form.getvalue(key)
+            if form[key].file:
```

Comment:
When parsing the form, make sure we aren't reading the file into d_form. d_form[file] is now an IO stream, not bytes.